src/demo/resources/configuration/scmspainads/PageNames.yml

```diff
@@ -0,0 +1,161 @@
+SCMSpainAds:
```

Comment:
Didn't we talk about removing these files?

![](https://media.giphy.com/media/l378BzHA5FwWFXVSg/giphy.gif)